test/extended/testdata/gssapi/fedora/base/fedora-base-pod

```diff
@@ -0,0 +1,38 @@
+{
+    "kind": "Pod",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "fedora-gssapi-base",
+        "namespace": "gssapiproxy"
+    },
+    "spec": {
+        "volumes": [
+            {
+                "name": "os",
+                "hostPath" : {
+                    "path": "OS_ROOT"
```

Comment:
@enj you have my sincere condolences... 

@liggitt I pushed him in that direction since there seems to be some understanding that extended tests can act as examples for use of the product, and that seems like the more appropriate way to do what he wanted using OpenShift